apiserver/src/controllers/ListController.ts

```diff
@@ -29,18 +29,13 @@ class ListController {
 	 * @function
 	 * @param {Request} req - Express request object
 	 * @param {Response} res - Express response object
-	 * @param {Function} next - Following middleware
 	 */
-	static async findMine(req: Request, res: Response, next: Function) {
-		try {
-			const { mine, shared } = (await List.getMine(req.database, req.uid || "")) as {
-				mine: string;
-				shared: string;
-			};
-			res.send({ mine, shared });
-		} catch (error) {
-			next(error);
-		}
```

Comment:
I don't know if it was really useful before but the trycatch block no longer exists ?

Reply:
try/catch was removed because it was useless before exactly.

Reply:
Try/catch removed because already useless exactly.